ale/drivers/nh_drivers.py

```diff
@@ -194,3 +190,66 @@ def exposure_duration(self):
           Exposure duration in seconds
         """
         return self.label['IsisCube']['Instrument']['ExposureDuration']
+
+
+class NewHorizonsMvicTdiIsisLabelNaifSpiceDriver(LineScanner, IsisLabel, NaifSpice, NoDistortion, Driver):
```

Comment:
Do MVIC TDI and MVIC Framer have the same type of distortion (legendre), or are they completely different?

Reply:
Seems like they do, the current drivers are not guaranteed to work in CSM or be accurate to the ISIS camera models. Currently, ISIS only gets it's ephemeris data from the ISDs and the specific distortion model doesn't matter in ISIS. This could easily change in the future but right now the distortion in the ISD doesn't change anything in ISIS

Reply:
Added the legendre distortion to the TDI driver